### PR TITLE
Fix styling for viewing individual member on admin side.

### PR DESCRIPTION
**Pull request by ivan-unfolds**

### CODE REVIEW

File: src/shared/dumb_components/field.js [hunk 1]
```diff
@@ -11,23 +11,23 @@ var Field = (props) =>
   ? <Input {...props} />
   : <div></div>
 
-var Display = ({ name, id, value, description }) =>
+var Display = ({ name, id, value, description, memberView }) =>
   <p>
     {description && name === 'News type: ' && <span><br /><i>{description}</i><br /></span>}
-    <br />
+    {memberView && <br />}
```
Comment (rjmk) [inline src/shared/dumb_components/field.js]: Hm. This really doesn't belong in a 'display' component, but the id (for example) is already coupled to the member fields in a similar way. I will have a think about this

Comment (rjmk) [inline src/shared/dumb_components/field.js]: We should drop the coupling in the `id`, allow a class to be passed in and the difference in spacing should be done in the CSS

**Comment by ivan-unfolds** [comment]
@rjmk Is this what you meant by decoupling `id`?

File: src/shared/dumb_components/field.js [hunk 2]
```diff
@@ -70,7 +69,7 @@ var make_select = (props, options) =>
 var make_placeholder = (name) =>
   name.match(/[dD]ate/) ? name.match(/[dD]ue/) ? 'dd/mm' : 'dd/mm/yyyy' : name
 
-var caser = (id, value) =>
-  id.match('email') ? value: to_title_case(value.replace(/-/g, ' '))
+var caser = value =>
```
Comment (rjmk) [inline src/shared/dumb_components/field.js]: I think let's pass `name` instead and still match `'email'`

File: src/shared/dumb_components/field.js [hunk 3]
```diff
@@ -11,23 +11,23 @@ var Field = (props) =>
...
-var Display = ({ name, id, value, description, memberView }) =>
-  <p>
-    {description && name === 'News type: ' && <span><br /><i>{description}</i><br /></span>}
-    {memberView && <br />}
-    <span className='info'>{name}</span>
-    <span id={'view-member-' + id}>
-      {caser(id, value)}
-    </span>
-  </p>
+var Display = ({ name, value, description, userViewClass }) =>
```
Comment (rjmk) [inline src/shared/dumb_components/field.js]: Why not just call this `className`?

File: src/shared/dumb_components/field.js [hunk 4]
```diff
@@ -11,23 +11,23 @@ var Field = (props) =>
...
+    <p>
+      <span className='info'>{name}</span>
+      <span>
+        {caser(value)}
+      </span>
+    </p>
+  </div>
 
-var Input = ({ className, name, options, touched, error, description, memberView, ...rest }) => {
+var Input = ({ userViewClass, name, options, touched, error, description, ...rest }) => {
```
Comment (rjmk) [inline src/shared/dumb_components/field.js]: Why the change in name?

**Comment by rjmk** [comment]
Yes, exactly!

File: src/shared/dumb_components/field.js (hunk 3 above)
Comment (ivan-unfolds) [inline src/shared/dumb_components/field.js]: I thought that we might need to know if it was a class for styling on admin view or on member view. Not sure if that makes sense if this is a display component. Have changed it back.

Comment (rjmk) [inline src/shared/dumb_components/field.js]: Yeah, this component shouldn't know that 'admin'/'member' views exist

File: src/styles/pages/_user-view.scss [hunk 5]
```diff
@@ -27,6 +27,18 @@ nav {
...
 
+.user-view-form {
+  padding: 0.5em 0;
+
+  p {
+    margin: 0;
+  }
+}
+
+.description {
```
Comment (rjmk) [inline src/styles/pages/_user-view.scss]: Should this be inside the `.user-view-form`?

Comment (ivan-unfolds) [inline src/styles/pages/_user-view.scss]: Yes makes sense.
